### ISSUE TITLE
Deploy and test a live demo

### ISSUE THREAD

**Issue by simonw** [simonw/datasette-hashed-urls#2]
The demo should run behind Cloudflare.

**Comment by simonw** [comment]
I'm trying a deploy direct to Vercel:
```
datasette publish vercel fixtures.db \
  --project datasette-hashed-urls \
  --install datasette-hashed-urls \
  --about 'datasette-hashed-urls demo' \
  --about_url 'https://github.com/simonw/datasette-hashed-urls/issues/2'
```

**Comment by simonw** [comment]
It crashed:
```
[ERROR] AttributeError: 'Datasette' object has no attribute '_hashed_url_databases'Traceback (most recent call last):  File "/var/task/vc__handler__python.py", line 293, in vc_handler    response = asgi_cycle(__vc_module.app, body)  File "/var/task/vc__handler__python.py", line 204, in __call__    loop.run_until_complete(asgi_task)  File "/var/lang/lib/python3.9/asyncio/base_events.py", line 642, in run_until_complete    return future.result()  File "/var/task/datasette_hashed_urls/__init__.py", line 38, in hashed_urls    if db_name in datasette._hashed_url_databases:
```
Presumably because `datasette-publish-vercel` doesn't arrange for the `startup()` hooks to be called!

https://github.com/simonw/datasette-publish-vercel/blob/b07f62d034e7cc62ba56309712ddb5c4349a38d7/datasette_publish_vercel/__init__.py#L39-L47

**Comment by simonw** [comment]
Now that I've shipped 0.2 I'm deploying the demo again:
```
datasette publish vercel fixtures.db \
  --project datasette-hashed-urls \
  --install datasette-hashed-urls \
  --about 'datasette-hashed-urls demo' \
  --about_url 'https://datasette.io/plugins/datasette-hashed-urls'
```
